### DISCUSSION TITLE
Fill in text in a Control Description?

### DISCUSSION THREAD

**Discussion by uhebeisen** [OpenOrienteering/mapper#2028] [Questions & Answers] [answered]
hello, I'm new to mapper, just starting to explore its features. Trying to draw a simple course on an already existing map. When doing the control description I have to draw all entries myself (right?). 
How can I write text (eg event name, course name, course data) into the rectangles at the top?

thank you for your help

**Comment by ghost** [comment, marked as answer]
> Trying to draw a simple course on an already existing map. When doing the control description I have to draw all entries myself (right?).

Here is short tutorial on «Course Design» using the Course Design Symbol Set:

- https://www.openorienteering.org/mapper-manual/pages/course_design.html

Also, in Mapper you may open symbol set `Course Design_10000.omap` (listed in main screen in "Tip of the day > Examples")

Alternatively, download file and open it as as map via "File > Open" 

> How can I write text (eg event name, course name, course data) into the rectangles at the top?

It is just simple, here is small screencast (hold <kbd>Shift</kbd> for "Point" and "Text" objects snapping to grid or to each other origins):
 
https://user-images.githubusercontent.com/13188580/150013875-87a8ff3f-02c3-4ac0-807d-8e45d341c812.mp4

**Reply by uhebeisen** [reply]
thanx, it is that easy :-)
